UPGRADE-6.8.md

```diff
@@ -1,4 +1,103 @@
 # 6.8.0.0
+## Introduced in 6.7.2.0
+## Removal of `EntityDefinition` constructor
+
+The constructor of the `EntityDefinition` has been removed, therefore the call of child classes to it need to be removed as well, i.e:
+```diff
+ <?php declare(strict_types=1);
+
+ namespace MyCustomEntity\Content\Entity;
+
+ use Shopware\Core\Content\Media\MediaDefinition;
+ use Shopware\Core\Content\Product\ProductDefinition;
+ use Shopware\Core\Framework\DataAbstractionLayer\EntityDefinition;
+
+ class MyCustomEntity extends EntityDefinition
+ {
+     // snip
+ 
+     public function __construct(private readonly array $meta = [])
+     {
+-        parent::__construct();
+         // ...
+     }
+ 
+     // snip
+ }
+```
+## Updated By Field is cleared on API updates
+In the next major version, the `UpdatedBy` field will be cleared when an object is updated via the API. This change ensures that the `UpdatedBy` field reflects the user who last modified the object through the API, rather than retaining the previous value.
```

Comment:
```suggestion
Now the `UpdatedBy` field will be cleared when an object is updated via the API.
This change ensures that the `UpdatedBy` field reflects the user who last modified the object through the API, rather than retaining the previous value.
```